docs/sources/alerting/set-up/configure-roles/index.md

```diff
@@ -0,0 +1,79 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/set-up/configure-roles/
+description: Configure roles and permissions for Grafana Alerting
+keywords:
+  - grafana
+  - alerting
+  - set up
+  - configure
+  - roles and permissions
+labels:
+  products:
+    - oss
+title: Configure roles and permissions
+weight: 150
+refs:
+  org-roles:
+    - pattern: /docs/grafana/
+      destination: https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/roles-and-permissions#organization-roles
```

Comment:
Or since they are the same destination, we can use a more general pattern prefix to catch both cases:

```suggestion
    - pattern: /docs/
      destination: https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/roles-and-permissions#organization-roles
```